src/cartridges/bm_adyen/cartridge/templates/default/adyenSettings/settingCards/adyenGivingSettings.isml

```diff
@@ -63,31 +68,36 @@
                <label class="form-title mb-0" for="backgroundFile">Background image for Adyen Giving component</label>
                <small id="backgroundImageHelo" class="form-text mb-1">Go to the classic page view on the top of this section to set up your Adyen Giving and upload the image there.<br>
                Once you save the configuration, the image will be automatically updated here.</small>
-               <div class="input-fields">
-                  <input type="text" class="form-control" name="AdyenGiving_backgroundUrl" id="backgroundFile" value="${AdyenConfigs.getAdyenGivingBackgroundUrl() || ''}" disabled>
+               <script>
+                  //Token and path can be found in settings.isml file
+                  window.adyenGivingBackgroundUrl = background_token +"#" + addition1 + addition4 + addition5;
+               </script>
+               <div class="wrapper-class "id="wrapper">
+                  <img src="${URLUtils.staticURL('icons/paperclip.svg')}" width="15" height="14" class="paperclip"/>
+                  <div class="drpbox-class" id="fileDropBoxCharitybackground" role="button">Click here to upload. </div>
+                  <div id="adyenGiving_background" class="form-group">
+                     <ul id="backgroundFileName"></ul>
+                  </div>
                </div>
-               <!-- <!input type="text" class="form-control" name="adyenGivingBackgrondUrl" id="backgroundFile" value="${AdyenConfigs.getAdyenGivingBackgroundUrl()}">
-                  <div class="wrapper-class "id="wrapper">
-                      <input id='charityBackgroundUrl' name="AdyenGiving_backgroundUrl" type='file' hidden/>
-                      <img src="${URLUtils.staticURL('icons/paperclip.svg')}" width="15" height="14" class="paperclip"/>
-                      <div class="drpbox-class" id="fileDropBoxCharitybackground" role="button">Drop files or click here to upload. </div>
-                  </div> -->
+               <br>
             </div>
             <div class="form-group">
                <label class="form-title mb-0" for="logoFile">Logo in Adyen Giving component</label>
                <small id="backgroundImageHelo" class="form-text mb-1">Go to the classic page view on the top of this section to set up your Adyen Giving and upload the image there.<br>
                Once you save the configuration, the image will be automatically updated here.</small>
-               <div class="input-fields">
-                  <input type="text" class="form-control" name="AdyenGiving_logoUrl" id="logoFile" value="${AdyenConfigs.getAdyenGivingLogoUrl() || ''}" disabled>
+               <script>
+                  //Token and path can be found in settings.isml file
+                  window.adyenGivingLogoUrl = logo_token +"#" + addition1 + addition4 + addition6;
```

Comment:
Can we join the different script tags that are scattered throughout this template?